### PR TITLE
Create more test cases for different error paths for the FAT

### PR DESCRIPTION
**Pull request by TayyabDev**
- Added more rigorous test cases for feature utility actions (installFeature and installServerFeatures)
- Add better error handling for edge cases in Feature Utility
- Add copyright headers

### CODE REVIEW

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 1]
```diff
@@ -779,6 +785,58 @@ ERROR_SINGLE_REPO_CONNECTION_FAILED=CWWKF1389E: Unable to create a JSON file at
...
+
+ERROR_IMPROPER_HTTPPROXY_FORMAT=CWWKF1400E: Invalid http_proxy environment variable format {0}. Ensure that the http_proxy variable is in the following format: http_proxy=http://<proxy-host>:<proxy-port>
+ERROR_IMPROPER_HTTPPROXY_FORMAT.explanation=Invalid proxy environment variable format.
+ERROR_IMPROPER_HTTPPROXY_FORMAT.useraction=Reformat the http_proxy environment variable to fit specifications, namely http_proxy=http://<proxy-host>:<proxy-port>.
+
+ERROR_IMPROPER_HTTPSPROXY_FORMAT=CWWKF1401E: Invalid https_proxy environment variable format {0}. Ensure that the https_proxy variable is in the following format: https_proxy=https://<user>:<password>@<proxy-host>:<proxy-port>
+ERROR_IMPROPER_HTTPSPROXY_FORMAT.explanation=Invalid proxy environment variable format.
+ERROR_IMPROPER_HTTPSPROXY_FORMAT.useraction=Reformat the http_proxy environment variable to fit specifications, namely https_proxy=https://<user>:<password>@<proxy-host>:<proxy-port>.
+
+ERROR_CANT_RESOLVE_FEATURE_PROVIDE_LINK=CWWKF1402E: Unable to obtain the following features: {0}. Ensure that the features are valid. Refer to the Open Liberty web page https://openliberty.io/docs/ref/feature for valid features.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]: Change
"Unable to obtain the following features: {0}. Ensure that the features are valid. Refer to the Open Liberty web page https://openliberty.io/docs/ref/feature for valid features."
To
"The following features were not obtained: {0}. Ensure that the features are valid. "

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
@Charlotte-Holt the recent change I made to the msg file just fixes a simple bug that occurred when getting the help messages. It is just a quick fix that removes a \n character.

**Comment by TayyabDev** [comment]
#build

**Comment by Charlotte-Holt** [comment]
@TayyabDev @herman-kailey Thanks for letting me know. LGTM. ID approved.

**Comment by TayyabDev** [comment]
#run-libby-bot

**Comment by TayyabDev** [comment]
The build failed due to an unrelated FAT failing. The feature utility FAT ran and passed.
